update-motd.d/99-wsl

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Warns if cloud-init didn't fully succeed during initialization of this instance.
+set -e
+
+stampfile="$HOME/.skip-cloud-init-warning"
+
+[ "$(systemd-detect-virt)" != "wsl" ] && exit 0
+[ -f "$stampfile" ] && exit 0
+
```

Comment:
We also need to test if /var/log/cloud-init.log exists prior to grepping it, otherwise grep will exit with an error.
It the file doesn't exist, then we just exit 0 without adding anything to motd.